packages/admin/cms-admin/src/userPermissions/hooks/currentUser.tsx

```diff
@@ -34,7 +35,25 @@ export const CurrentUserProvider: React.FC<{
         }
     `);
 
-    if (error) throw error.message;
+    // As this Provider is very high up in the tree, don't rely on ErrorBoundary or a configured intl-Provider here
```

Comment:
We could change the provider order IMO. The user provider would need to be inside the ThemeProvider, otherwise the theme wouldn't be used